aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -667,10 +670,9 @@ Tensor std_var_common_impl_mps(
                            input_shape,
                            axes);
       num_output_dims = num_input_dims;
-      output_shape = (int64_t *)malloc(num_output_dims  * sizeof(int64_t));
       for (int i = 0; i < num_input_dims; i++)
       {
-          output_shape[i] = (int64_t) 1;
+          output_shape.push_back((int64_t) 1);
```

Comment:
Hmm, it would be nice to add an assert that `output_shape` is empty before entering the loop (can do it as followup PR)